catalog/transformer/catalog-transformer-html/src/main/resources/templates/template.hbs

```diff
@@ -235,6 +241,38 @@
             attribute.parentElement.style.display = "";
           }
         }
+
+        updateTableRowColors();
+      }
+      function isEmptyValueAttribute(attribute) {
```

Comment:
nit: might be better name hasEmptyAttributeString or hasEmptryAttributeValue

Reply:
I prefer "is" here because this function checks if it is a empty valued attribute not if it has an empty valued attribute. 

In the context of a metacard a method like `hasAttribute()` would make sense to check if a metacard has an attribute whereas a method like `isEmptyAttribute()` would check something more specific.